### ISSUE TITLE
Can't see all players in Player details

### ISSUE THREAD

**Issue by sakurinha** [ebshimizu/stats-of-the-storm#102]
I can only see 15 of the 27 players that participated in the replays, the 15 that played the most

**Comment by ebshimizu** [comment]
Yep this is actually functioning as normal. To fix, go to the settings page and change the "Player Match Threshold" limit to a lower value (the default is 5), and click the "set threshold" button. Changing it to 1 should show all players.

This was done to help with the responsiveness of larger databases. I think I added this recently and didn't update the docs page yet.
